### ISSUE TITLE
Close all apps + lock icons and widgets 

### ISSUE THREAD

**Issue by NathanPercheron** [Archfx/duo-de#122]
### Acknowledgements

- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] This is not a feature request.
- [X] This issue is not a duplicate of an existing issue.
- [X] The issue is solely related to this GSI.
- [X] This issue occurs with the latest available build.
- [X] I am not running a custom vendor on my device.
- [X] This issue is not kernel related.
- [X] This issue is not related to Magisk/KernelSU/APatch.

### Info

- Device: surface duo 2
- GSI version: treble_arm64_bgN-userdebug 15 AP3A.241105.008 eng.aruna.20241129.203334
- 
- Vendor version: 
ARCHFX/DUO-DE A15 (archfx.github.io/duo-de)

### Expected Behavior

When I slide my finger from the bottom to the top with a hold to display the open applications and be able to close them by sliding from the right to the left To reach the "close all applications" button All with the option to lock icons and widgets enabled.

### Current Behavior

When I perform the action of showing already opened applications to be able to close them with the icon lock option enabled, I can display the applications But once I slide my finger from left to right, I get a screen freeze for a few seconds and it closes 

### Possible Solution

To work around the problem I have to disable the option to lock icons and widgets Which is annoying because I encountered a problem with the icons being reset to default from time to time. 

### Steps to Reproduce

1. Enable the option to lock icons and widgets 
2. Show apps open in background tasks
3. Slide your finger to reach the button to close all applications

**Comment by wilhelmstroker** [comment]
Yep can confirm I'm seeing the same issue when having the screen layout locked. Looks like the launcher crashes with that setting enabled.

**Comment by Archfx** [comment]
This was an issue with the Trebuchet launcher (from LineageOS), [new beta releases](https://github.com/Archfx/duoTreble/releases/tag/v2025.01.08) are based on Launcher3 (from Android). Let's hope this and #84 will be fixed soon. The current beta build has a couple of issues that prevent it from daily usage.

**Comment by Ethanol10** [comment]
Closing this issue as it's a duplicate of the existing prior issue https://github.com/Archfx/duo-de/issues/84
